phpBB/styles/prosilver/theme/content.css

```diff
@@ -383,8 +383,7 @@ dl.faq dt {
 }
 
 .content ul, .content ol {
-	margin-bottom: 1em;
-	margin-left: 3em;
+	margin: 1em 0 1.2em 3em;
```

Comment:
I just checked and changing it to this creates the exact same margins as the codebox has in Chrome:
```margin: 0.8em 0 0.9em 3em;```

I'd prefer if the resulting margins would be the same.